frontend/test/get_application_url_test.dart

```diff
@@ -44,14 +60,21 @@ void main() {
       final String? applicationUrlQueryKeyReferenceNumber = null;
       final cardInfo = CardInfo()
         ..fullName = 'Max Mustermann'
-        ..expirationDay = 365 * 40 // Equals 14.600
+        ..expirationDay =
+            365 *
+            40 // Equals 14.600
```

Comment:
```suggestion
        ..expirationDay =
            14600 // 365 * 40
```
a little bit better this way